### DISCUSSION TITLE
Widgets einzeln transparent setzen

### DISCUSSION THREAD

**Discussion by ghost** [osmandapp/OsmAnd#18808] [Ideas]
Ein Vorschlag: Es wäre hilfreich, wenn sich die Widgets (links, rechts, oben und unten) separat transparent schalten ließen. Ich persönlich würde oben und unten nicht transparent schalten. Dadurch könnte man z. B. den Straßennamen oben besser ablesen, ohne dass es die Kartendarstellung stört. 

Rechts und links würde ich transparent setzen, um in der Karte nicht zu viele Informationen abzudecken.

**Comment by ghost** [comment]
Github hat meinen Text durcheinandergewürfelt. Deshalb nochmal:

Es wäre hilfreich, wenn man Widgets separat transparent setzen könnte. Dann würde ich oben und unten nicht transparent setzen, da man Informationen so besser lesen kann. Es stört aber die Kartendarstellung nicht. Die Widgets links und rechts stehen allerding direkt in der Kartendarstellung. Hier ist es hilfreich, wenn man die Widgets transparent setzen kann, um die Dartsellung nicht zu stören.

**Reply by ghost** [reply]
Ich bin neu in Github. Mein geschriebener Text wird zerstückelt. Ist das normal?

**Reply by DmitryAlexei** [reply]
Please write in English so that community members can easily understand your suggestions.
I suppose it is a duplicate of https://github.com/osmandapp/OsmAnd/issues/16017 and similar issues mentioned in that ticket.

**Comment by ghost** [comment]
It would be helpful if widgets could be made transparent separately. Then I wouldn't put the top and bottom transparently, because you can read information better that way. However, it does not interfere with the map display. However, the widgets on the left and right are located directly in the map display. Here it is helpful if you can set the widgets transparently so as not to disturb the display.
